### DISCUSSION TITLE
Training is suddenly much faster.

### DISCUSSION THREAD

**Discussion by hbqdev** [d8ahazard/sd_dreambooth_extension#732] [General]
Did anyone notice the same thing?
I'm on the 3090.

**Comment by 1025307336** [comment]
*2,But the problem is that the training effect is very poor

**Reply by ExponentialML** [reply]
**Edit** This is now merged, so you can just update the extension.

Hey @1025307336 .
Could you test the changes here in this pull request? https://github.com/d8ahazard/sd_dreambooth_extension/pull/757/commits/001836d1b482a65940a794bff5ee09f55e4487e6
Simply change the `.yaml` file after converting the checkpoint `models/dreambooth/*your_model_folder*`

1. Find the `.yaml` file (will be the same name as the `model folder.yaml`).
2.  Make the changes like the link above to `your_model.yaml` file (red = old, blue = new).
3. Restart the entire webui to make sure the model is reloaded in the Dreambooth tab correctly.

If it does work, then you can change all three of those `.yaml` files to match in your `extensions/config/*.yaml` directory to match for the time being. That way they're automatically created when you have a new checkpoint.

Report back and let us know if the results are better, thanks!

**Comment by rpgsolomaker** [comment]
> Did anyone notice the same thing? I'm on the 3090.

Would you mind sharing the settings that you used? And how many images were in your dataset directory and classification folder? Ty

**Reply by 1025307336** [reply]
I've found out why，His counting algorithm in cmd has changed，steps=bs * Normal running speed，The bs setting may have been disabled so it didn't slow down the computation
